tests/js/server/replication/fuzz/replication-fuzz-global.js

```diff
@@ -118,14 +122,18 @@ function ReplicationSuite() {
       while ((lastLogTick !== state.lastLogTick) && (count < 500)) {
         lastLogTick = 0;
         count += 1;
-        db._collections().forEach(col => {
-          if (col.name()[0] !== '_') {
-            lastLogTick += col.count() + 1;
-          }
+        db._flushCache();
+        db._useDatabase('_system');
+        db._databases().forEach(function(d) {
+          db._useDatabase(d);
+          db._collections().forEach(col => {
+            if (col.name()[0] !== '_') {
+              lastLogTick += col.count() + 1;
+            }
+          });
```

Comment:
After iteration over all available databases, we may end up being in a different database than before. Should we add an `db._useDatabase('_system');` after this loop, or even go back to the database that we left off in?